src/pages/index.js

```diff
@@ -2,19 +2,21 @@ import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import PopupWithImage from "../components/PopupWithImage.js";
-import PopupWithConfirm from "../components/PopupWithConfirm.js"; // Import the PopupWithConfirm class
+import PopupWithConfirmation from "../components/PopupWithConfirmation.js";
 import UserInfo from "../components/UserInfo.js";
 import Section from "../components/Section.js";
-import { initialCards, validationOptions } from "../utils/constants.js";
+import { validationOptions, initialCards } from "../utils/constants.js";
 import "../pages/index.css";
+import api from "../components/Api.js";
 
 document.addEventListener("DOMContentLoaded", () => {
   const profileEditButton = document.querySelector("#profile-edit-button");
   const addCardButton = document.querySelector("#add-card-button");
-
```

Comment:
Needs correction: When I reload the page I see the default cards, but there should be the cards received from the server